Makefile

```diff
@@ -6,7 +6,7 @@ setup: content
 
 content:
 	rm -rf content assets/static/images
-	git clone --branch master --single-branch --depth 1 https://github.com/elixirschool/elixirschool.git content
+	git clone --branch fix-post-image-links --single-branch --depth 1 https://github.com/elixirschool/elixirschool.git content
```

Comment:
I will undo this change before I mark this pr as ready to review - I am just checking the fix for https://github.com/elixirschool/school_house/issues/145